cypress/integration/create.js

```diff
@@ -212,4 +232,45 @@ describe('Create Page', () => {
             expect(el).to.have.value('The real Slim Shady!')
         );
     });
+
+    it('should not show rich text input error message when field is untouched', () => {
+        cy.get('.ra-input-body .ra-rich-text-input > p').should('not.exist');
+    });
+
+    it('should show rich text input error message when form is submitted', () => {
+        CreatePage.submit();
+        cy.get('.ra-input-body .ra-rich-text-input > p').should('exist').contains('Required');
+    });
+
+    it.only('should not show rich text input error message when form is submitted and input is filled with text', () => {
```

Comment:
only spotted

Reply:
Good catch!
BTW, is there better way to run single test in cypress?
Using `.only` has a drawback like this one

Reply:
None that I'm aware of. You miss jest too ? :wink:

Reply:
Trying to use https://github.com/bahmutov/cypress-skip-and-only-ui , will submit a PR if it works as declared :)